conf/spark-env.sh.template

```diff
@@ -23,6 +23,7 @@
 # Options read when launching programs locally with
 # ./bin/run-example or ./bin/spark-submit
 # - HADOOP_CONF_DIR, to point Spark towards Hadoop configuration files
+# - YARN_CONF_DIR, to point Spark towards YARN configuration files
```

Comment:
When you use ‘spark-submit --master yarn /home/...jar’  to submit the spark job, you need to set the YARN_CONF_DIR value.

Can i modify it like this? as follows:
+# - YARN_CONF_DIR, to point Spark towards YARN configuration files **when you use YARN.**